src/zos.cc

```diff
@@ -2851,8 +2851,19 @@ int futimes(int fd, const struct timeval tv[2]) {
   memset(&atr, 0, sizeof(atr));
   atr.att_mtimechg = 1;
   atr.att_atimechg = 1;
-  atr.att_atime = tv[0].tv_sec;
-  atr.att_mtime = tv[1].tv_sec;
+  if (tv == NULL)
```

Comment:
Can you change it so it follows the existing style:
```
if (tv == NULL) {
```

Instead of:
```
if (tv == NULL)
{
```

Reply:
I have corrected it, Igor! Thank you!